modules/organism.js

```diff
@@ -46,7 +46,7 @@ Organism.prototype = {
         var food = this.nearByFood[i];
         if (!food.depleted)
         {
-          this.follow(food.location, food.radius, food.mass * 10);
+          this.follow(food, food.radius, food.energy/(Constant.ENERGY*2));
```

Comment:
we should add
`food.mass = food.energy / Constant.ENERGY`
in Food.js, and use
`food.mass / 2`